### PR TITLE
Merge dev branch to main

### PR DESCRIPTION
**Pull request by yd24**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,9 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.3.4",
```
Comment (KamWatts) [inline package.json:9]: Good job importing axios into your data

File: package.json [hunk 2]
```diff
@@ -6,7 +6,9 @@
...
     "react": "^18.2.0",
+    "react-bootstrap": "^2.7.2",
```
Comment (KamWatts) [inline package.json:11]: You have react-bootstrap, but I notice you did not install (vanilla) bootstrap

File: src/Main.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+class Main extends React.Component {
+    render() {
+        return;
```
Comment (KamWatts) [inline src/Main.js]: Did you end up using the Main component for anything? Or is this pending for further design improvements?

File: src/App.js [hunk 4]
```diff
@@ -1,24 +1,15 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
```
Comment (KamWatts) [inline src/App.js:8]: Good job condensing your files below for import. Takes away from all the noise in your parent file.

File: src/Footer.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```
Comment (KamWatts) [inline src/Footer.js:1]: Next time, I will use this technique of splitting the components up this way. Good job. Makes your file structure easy to navigate.

File: package.json (hunk 2 above)
Comment (yd24) [reply package.json:11]: Good catch, I have added it in the latest commits.

File: src/Main.js (hunk 3 above)
Comment (yd24) [reply src/Main.js]: Yes, this was just boilerplate. Once the children components were made, I put them in Main.

**Pull request merged**